consensus/ibft/ibft.go

```diff
@@ -866,6 +896,21 @@ func (i *Ibft) runAcceptState() { // start new round
 				continue
 			}
 
+			if i.bridge != nil {
+			StateTxValidationLoop:
+				for _, tx := range block.Transactions {
+					if tx.Type != types.TxTypeState {
+						continue
+					}
+
+					if err := i.bridge.ValidateTx(tx); err != nil {
+						i.logger.Error("block has the invalid state transaction", "tx", tx.Hash, "err", err)
+						i.handleStateErr(errBlockVerificationFailed)
+						break StateTxValidationLoop
+					}
+				}
+			}
```

Comment:
Please refactor this part of the code to not use labels, there is nothing elegant with writing code like it's the 1970s

Reply:
Removed when moving this flow to hook 11e272b3658a425586dc95f8ba34bab2ec9cc563